ddl/delete_range.go

```diff
@@ -283,7 +283,7 @@ func insertJobIntoDeleteRangeTable(ctx sessionctx.Context, job *model.Job) error
 					return errors.Trace(err)
 				}
 			}
-			return nil
+			// Do not return, because global indexes entries begin with talbeID.
```

Comment:
The impact on performace of delete range is not negligible (tikv raftstore membership change, rocksdb handle for overlap range and compaction handle so on), so it's better to check the table actually use global index, to avoid the unnecessary delete range job.